contracts/account/modules/ERC7579ECDSAValidator.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {ECDSA} from "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import {ERC7579Validator} from "./ERC7579Validator.sol";
+
+/**
+ * @dev {ERC7579Validator} for {ECDSA} signatures.
+ */
+abstract contract ERC7579ECDSAValidator is ERC7579Validator {
+    mapping(address sender => address signer) private _associatedSigners;
```

Comment:
would it make sens that a sender supports multiple signers ?
```solidity 
mapping(address sender => mapping(address signer => bool isOperator))
```

We may even want to make that a N out of M multisig 

Beyond the "choice of cryptographic primitive", there are a lot of relevant validator that we may want to implement in terms of behavior. I'd leave that to a future PR (this one is already big with just the Account part)

Reply:
Would it be an acceptable middle ground to remove the `contracts/accounts/modules` from this PR (or move the necessary one to the mocks) and open PRs for these after we merge this one (with "just" the `contracts/accounts/extensions` ?)